config/webpack.common.js

```diff
@@ -211,15 +211,6 @@ const config = [
 			poll: 1000,
 		},
 	},
-	{
-		entry: {
-			'eventespresso-core-css-default': pathToEDTRv1 + 'components/ui/styles/themes/default/index.js',
-		},
-		module: moduleConfigWithJsAndCssRules,
-		watchOptions: {
-			poll: 1000,
-		},
-	},
```

Comment:
this file will completely/ safely removed in https://github.com/eventespresso/event-espresso-core/pull/2609